rebar.config

```diff
@@ -40,7 +41,7 @@
               ]}.
 
 {plugins, [
-           {grpcbox_plugin, {git, "https://github.com/andymck/grpcbox_plugin.git", {branch, "andymck/ts-master"}}},
+           {grpcbox_plugin, {git, "https://github.com/andymck/grpcbox_plugin.git", {branch, "andymck/ts-master/combined-opts-and-template-changes"}}},
```

Comment:
i'm not clear if this is even necessary anymore or if there's anything else that might be using it.

Reply:
I've wondered about this.. did you try removing it?